vpr/src/place/place.cpp

```diff
@@ -1466,7 +1497,26 @@ static e_move_result try_swap(float t,
         /* 1 -> move accepted, 0 -> rejected. */
         move_outcome = assess_swap(delta_c, t);
 
-        if (move_outcome == ACCEPTED) {
+        if (manual_move && manual_move_info->valid_input) {
```

Comment:
Move the manual_move code into a subroutine so the control routine (here) is simpler.